src/runtime_src/core/common/thread.cpp

```diff
@@ -140,8 +140,47 @@ set_thread_policy(std::thread&)
 }
 
 static void
-set_cpu_affinity(std::thread&)
+set_cpu_affinity(std::thread& thread)
 {
+  static bool initialized = false;
+  static DWORD_PTR affinityMask = 0;
```

Comment:
user space code never uses camelCase; of course windows functions are CamelCase, but variables we create should not be.